### DISCUSSION TITLE
Ability to disable instantaneous teleportation to the nether for players in creative mode.

### DISCUSSION THREAD

**Discussion by shideus** [PurpurMC/Purpur#1048] [__Completed__] [answered]
Hi! 👋 

It would be cool to be able to disable instant teleportation to the nether in the purpur config for creative players. So that teleportation occurs with a delay, as in survival. Many players are very annoyed by instant teleportation, it is not always convenient.

**Comment by granny** [comment, marked as answer]
Starting with build #1975, there are now two new options in the world config; `player.portal-wait-time` and `player.creative-portal-wait-time`. Giving a player the `purpur.portal.instant` will ignore whatever is set in these two options.
